Index: src/gpasman4.sample
===================================================================
--- src/gpasman4.sample
+++ src/gpasman4.sample	2010-01-03 14:50:00.914386762 +0100
@@ -6,4 +6,3 @@
 customer=customer 3; group=Group A; server=srvE; service=ssh; serviceplus=port 322; user=root; password=pass6
 customer=customer 3; group=Group A; server=srvF; service=ssh; serviceplus=port 322; user=root; password=pass7
 customer=customer 3; group=Group A; server=srvF; service=http; serviceplus=http://localhost/; user=beuarg; password=gnapgnap
-

